includes/class-internal-events.php

```diff
@@ -132,30 +145,47 @@ public function force_publish_missed_schedules() {
 	public function confirm_scheduled_posts() {
 		global $wpdb;
 
-		$future_posts = $wpdb->get_results( $wpdb->prepare( "SELECT ID, post_date FROM {$wpdb->posts} WHERE post_status = 'future' AND post_date > %s LIMIT 25;", current_time( 'mysql', false ) ) );
+		$page     = 1;
+		$quantity = 25;
+
+		do {
```

Comment:
Same as above; the loop is probably not necessary. If we expect lots of posts, can we just bump the limit to 100 or something similar (especially since we're not fetching a lot of data)?